DuckDuckGo/MainViewController.swift

```diff
@@ -1271,22 +1271,17 @@ class MainViewController: UIViewController {
 
         Task {
             do {
-                try await NetworkProtectionCodeRedemptionCoordinator().exchange(accessToken: token)
-                print("[NetP Subscription] Exchanged access token for auth token successfully")
+                try NetworkProtectionKeychainTokenStore().store(NetworkProtectionKeychainTokenStore.makeToken(from: token))
```

Comment:
Seems like quite a lot of knowledge for the MainViewController to hold. How about hiding the token conversion within the token store implementation?